### PR TITLE
Define CMS_JobType and TaskType for analysis jobs

### PR DESCRIPTION
**Pull request by stiegerb**
Take CMS_JobType from ad also for analysis jobs; use same value for TaskType

### CODE REVIEW

File: src/htcondor_es/convert_to_json.py [hunk 1]
```diff
@@ -588,12 +588,12 @@ def convert_to_json(ad, cms=True, return_dict=False, reduce_data=False):
         else:
             camp2 = ttype
 
-        result['CMS_JobType'] = str(ad.get('CMS_JobType', 'Analysis' if analysis else 'Unknown'))
+        result['CMS_JobType'] = str(ad.get('CMS_JobType' 'Unknown'))
```
Comment (amaltaro) [inline src/htcondor_es/convert_to_json.py]: You missed a comma "," here. How about making it backwards compatible for the moment (until CRAB jobs start showing up with this CMS_JobType ad too)?

Comment (stiegerb) [reply src/htcondor_es/convert_to_json.py]: We're stupid. This was already correct (and backwards compatible) as it was. If `'CMS_JobType'` is filled, it's already taken. Only if it's not filled we overwrite it with `"Analysis"`.

**Comment by belforte** [comment]
:-(
blame it on me. I owe beer to everybody who cares to ask

File: src/htcondor_es/convert_to_json.py (hunk 1 above)
Comment (amaltaro) [reply src/htcondor_es/convert_to_json.py]: LoL, you're right!

**Comment by stiegerb** [comment]
Ok, so I think the only thing we have to change is filling TaskType with CMS_JobType for analysis job. I updated the commit accordingly.

**Comment by belforte** [comment]
I still think that best approach is that TaskType is set by the submitting tool
in a classAd, not guessed in this script. The less we massage data at this point
in the flow, the better off we are.
Maybe use same logic as for famous CMS_JobType so that when CMS_TaskType ad is defined,
it is used.

**Comment by stiegerb** [comment]
This change is obsolete now with #64 merged.

**Pull request closed without merging**